themes/scss/map/overlays.scss

```diff
@@ -504,3 +504,10 @@ $index-items: 7 6 5 4 3 2 1;
 .Legend-placeholderInner {
   padding: 16px 24px 0;
 }
+
+
+@media (max-width: 759px) {
+  .CDB-Legends-canvas {
+    display: none !important;
```

Comment:
not because we are adding display:block inline in the html
![image](https://cloud.githubusercontent.com/assets/1214201/19191087/2695261c-8ca1-11e6-9376-4035c26a3cba.png)